charts/snowplow-iglu-server/templates/iglu-ingress.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if eq .Values.global.cloud "azure" }}
```

Comment:
Given the want to make traefik a more common pattern everywhere is it worth instead just adding traefik support in general that is not restricted to a particular cloud?

Reply:
Yes that unblock this feature to all platforms that support Traefik. So there is no Azure block required for special settings.

Reply:
Configs updated